src/bio/bio_wal.c

```diff
@@ -7,6 +7,9 @@
 
 #include "bio_wal.h"
 
+#include <gurt/telemetry_common.h>
+#include <gurt/telemetry_producer.h>
```

Comment:
These two headers looks not necessary to me.

Reply:
and this..

Reply:
Done. Thanks!